sphinx/screens-detail/create-tribe/create-tribe/src/main/java/chat/sphinx/create_tribe/ui/CreateTribeFragment.kt

```diff
@@ -94,7 +105,7 @@ internal class CreateTribeFragment: SideEffectFragment<
                 if (hasFocus) {
                     return@setOnFocusChangeListener
                 }
-                editTextTribeImage.text.toString()?.let { imageUrl ->
+                editTextTribeImage.text.toString().let { imageUrl ->
```

Comment:
calling `.toString()` on an `EditText.text` value will result in the use of the `Any?.toString()` extension function. this will produce a string value of `"null"` if the `EditText.text` is null. Utilizing the null check operator `?`, as in `EditText.text?.toString()` will **not** use the `Any?.toString()` extension function, allowing for proper use of the `?.let { nonNullValue -> }`

This `Any?.toString()` usage is also present in the `setupCreateTribe` methods